test/addons/rbd-mirror/test

```diff
@@ -56,11 +56,7 @@ def rbd_mirror_image_status(cluster, image):
 
 def test_volume_replication(primary, secondary):
     print(f"Deploying pvc {PVC_NAME} in cluster '{primary}'")
```

Comment:
This log deploy all tests resources, you can update to to describe what is does. See addons/rook-ceph/test for example.

Reply:
Done